boltdesignsystem/bolt
# WWW-218: Microjourney Feedback button, WWW-191: Calc Input Update

## Jira

- https://pegadigitalit.atlassian.net/browse/WWW-218
- https://pegadigitalit.atlassian.net/browse/WWW-191

## Summary

- Removed feedback button
- Added input pattern to the calculator form

## Details

- Removed the feedback button from the micorjourney Admin expe…rience
- Added a pattern to the calculator input fields to remove the addition of a decimal

## How to test
- Pull down this branch
- Navigate to "/pattern-lab/?p=experiments-step-2-demo"
- Confirm there is no "Feedback" button showing on the bottom right of the screen
